lib/components/narrative/metro/default-route-renderer.tsx

```diff
@@ -26,7 +25,7 @@ const Block = styled.span<{ color: string; isOnColoredBackground?: boolean }>`
 `
 
 export type RouteRendererProps = {
-  leg: Leg & { onColoredBackground?: boolean }
+  leg: Leg & { onColoredBackground?: boolean; origColor: string }
```

Comment:
```suggestion
  leg: Leg & { onColoredBackground?: boolean; origColor?: string }
```

`origColor` is optional!